tools/run_tests/run_performance_tests.py

```diff
@@ -450,15 +452,23 @@ def finish_qps_workers(jobs):
 if not scenarios:
   raise Exception('No scenarios to run')
 
+total_scenario_failures = 0
+total_jobs_killed = 0
 for scenario in scenarios:
   if args.dry_run:
     print(scenario.name)
   else:
     try:
       for worker in scenario.workers:
         worker.start()
-      jobset.run([scenario.jobspec,
-                  create_quit_jobspec(scenario.workers, remote_host=args.remote_driver_host)],
-                 newline_on_success=True, maxjobs=1)
+      scenario_failures, _ = jobset.run([scenario.jobspec,
+                                 create_quit_jobspec(scenario.workers, remote_host=args.remote_driver_host)],
+                                 newline_on_success=True, maxjobs=1)
+      total_scenario_failures += scenario_failures
     finally:
-      finish_qps_workers(scenario.workers)
+      # Consider jobs that need to be killed as failures
+      total_jobs_killed += finish_qps_workers(scenario.workers)
+
+if num_failures > 0:
+  print(str(total_scenario_failures) + " scenarios failed and " + str(total_jobs_killed) + " jobs killed")
```

Comment:
nit:
`print ("%s scenarios failed and %s qps worker jobs killed" % (total_scenario_failures, total_jobs_killed))`  is much better in python. There's also a new syntax for doing this (preferred by some).

nit : "jobs killed" is a bit ambiguous. You are talking about  qps worker jobs here but the scenarios are also jobs.

Feel free to merge and address this later.